core-services/ci-secret-bootstrap/_config.yaml

```diff
@@ -2675,6 +2675,15 @@ secret_configs:
   - cluster: app.ci
     name: github-deploymentconfig-trigger
     namespace: ci
+- from:
+    app.ci.config:
+      attachment: sa.endurance.app.ci.config
+      bw_item: endurance
```

Comment:
@alvaroaleman not sure what "bw_item" is supposed to be, can you sanity check this?

i'm trying to copy the SA kubeconfig from vault into the build farm clusters.